docs/whats-new/index.yml

```diff
@@ -0,0 +1,35 @@
+### YamlMime:Landing
+
+title: .NET Documentation What's New?
+summary: "Welcome to What's new in .NET docs. Quickly find the latest changes in .NET docs."
+
+metadata:
+  title: .NET Documentation What's New?
+  description: "Welcome to What's new in .NET docs."
+  ms.topic: landing-page
+  ms.date: 12/12/2019
+
+landingContent:
+  - title: "What's new"
+    linkLists:
+      - linkListType: whats-new
+        links:
+          - text: November 2019
+            url: 2019-11.md
+
+  - title: "Open source contributions"
```

Comment:
```suggestion
  - title: Contribute
```

Reply:
I went with different text, based on Tom's comment. I think it addresses the same concern.